packages/api/test/user.spec.js

```diff
@@ -229,11 +303,28 @@ describe('GET /user/uploads', () => {
     })
     assert(res.ok)
 
-    const expected = [userUploads[0]]
     const link = res.headers.get('Link')
     assert(link, 'has a Link header for the next page')
-    assert.strictEqual(link, `</user/uploads?size=${size}&before=${encodeURIComponent(expected[0].created)}>; rel="next"`)
+
+    assert.strictEqual(link, `</user/uploads?size=${size}&offset=${size}>; rel="next"`)
+    const uploads = await res.json()
+    const expected = [userUploads[0]]
+    assert.deepStrictEqual(uploads, expected)
+  })
+
+  it('does not paginate when all results are returned', async () => {
+    const token = await getTestJWT()
+    const size = 1000
+    const res = await fetch(new URL(`/user/uploads?size=${size}`, endpoint).toString(), {
+      method: 'GET',
+      headers: { Authorization: `Bearer ${token}` }
+    })
+    assert(res.ok)
+
+    assert.notEqual('link' in res.headers, 'does not have a Link header for the next page')
```

Comment:
I'm guessing this is a typo and should be `assert.not` (or similar)?

Reply:
Looked at this, `.not` is not available when using node's `assert()`. I changed this to be `.notStrictEqual` as `.notEqual` [seems to be legacy](https://nodejs.org/api/assert.html#assertnotequalactual-expected-message). 

I could actually simply inverse the bool with `!` and just `assert(!'link' in res.headers)` it..
But I like how declarative `notEqual` is and how a failure is reported.. then again that's what that optional message field is for.🤔 
Am open to suggestion on how best to do this!

Reply:
I was just meaning that currently you're not supplying a message, you're comparing the boolean result of `'link' in result.headers` to the string `'does not have a Link...'`, so they will never be equal, regardless of whether `'link'` is in `res.headers` or not!

Reply:
Ops, yep that's a great spot, my bad! Changed this to a `.ok` for now, thanks!